Libplanet.Stun/Libplanet.Stun.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
```

Comment:
Could we configure the same analyzers to Libplanet project?  See also *Libplanet.csproj* file.